### ISSUE TITLE
i found many backticks in password simple，is it right？

### ISSUE THREAD

**Issue by Explorer1092** [brannondorsey/PassGAN#6]
root@07d198dcd43f:/notebooks# tail  /mnt/passgan/samples/samples_172900.txt 
Baugedd*76
ashas#1```
r.macj````
AvirTAbN``
dice@35```
Salderr65`
Ml4pann```
Ep044j8```
s?j009````
koph@9306`

**Comment by brannondorsey** [comment]
Yes, backticks are used to pad passwords to a fixed length (10 chars). Replace them with null once generated :+1:

**Comment by brannondorsey** [comment]
You found them in all generated passwords with length < 10 correct?

**Comment by Explorer1092** [comment]
yes，length < 10。
Should I replace these backticks by myself?

**Comment by brannondorsey** [comment]
Unfortunately, yes. I didn't realize I left the backticks in there :fearful:. Away from my dev machine until next week, but I'll update the repo to fix this then. Until then, something like this should work:

```python
# save this as remove-backticks.py
with open('samples.txt', 'r') as f:
    for line in f:
        print(line.replace('`', ''))
```

```bash
# run it like this
python remove-backticks.py > cleaned-samples.txt
```

**Comment by brannondorsey** [comment]
Actually, ya know what, this [shouldn't even be happening](https://github.com/brannondorsey/PassGAN/blob/master/sample.py#L97). Not sure what is up. I'll take a look next week, thanks!

**Comment by tjw960414** [comment]
您好，我在运行train.py文件时显示错误output-o需要参数，可是我不知道在哪加，请您解答感谢

**Comment by brannondorsey** [comment]
@tjw960414, are you referencing this issue or do you have a new question? If this is unrelated would you mind opening a new issue and providing more information about your problem?

Thanks.
